docs/dispatcher/dependency_injection.rst

```diff
@@ -0,0 +1,56 @@
+######
+Dependency injection
```

Comment:
I don't think so this is a clear dependency injection mechanism, but i don't know another name for this feature

Reply:
> I don't think so this is a clear dependency injection mechanism
The basic concept of dependency injection is clearly visible in the example of passing a router to the current handler (by "event_router" keyword) - the handler itself depends on it and has access to it. 

But you really can pass anything through the handling context and do it both at application startup and at runtime. Nevertheless, I saw other libraries that also name the concept of passing arguments to the function by parameter names / type hints like "depends".

> but i don't know another name for this feature
Maybe "Handling context"?